base/src/main/java/io/spine/type/MessageType.java

```diff
@@ -391,5 +393,23 @@ public boolean isUuidValue() {
         FieldName uuid = FieldName.of("uuid");
         boolean nameMatches = uuid.equals(theField.name());
         boolean typeMatches = theField.isString();
-        return nameMatches && typeMatches;    }
+        return nameMatches && typeMatches;
+    }
+
+    /**
+     * Checks if the message has the given option.
+     *
+     * <p>The option must be known at runtime. An uninterpreted option cannot be found this way.
+     *
+     * @param optionName
+     *         name of the option
+     * @return {@code true} if this type is marked with an option with the given name,
+     *         {@code false} otherwise
+     */
+    public boolean hasOption(String optionName) {
+        MessageOptions options = descriptor().getOptions();
+        Set<FieldDescriptor> presentOptions = options.getAllFields().keySet();
+        return presentOptions.stream()
+                             .anyMatch(op -> op.getName().equals(optionName));
```

Comment:
Assuming the previous request for this method, please flip equals. It would read a bit simpler.